docs/coordinates/index.rst

```diff
@@ -356,16 +356,16 @@ interface to these objects as well as various convenience methods and
 more string-parsing capabilities.
 
 Separating these concepts makes it easier to extend the functionality of
-`~astropy.coordinates`.  It allows representations, frames, and
+`~astropy.coordinates`. It allows representations, frames, and
 transformations to be defined or extended separately, while still
-preserving the high-level capabilities and simplicity of the |skycoord|
+preserving the high-level capabilities and efficiency of the |skycoord|
```

Comment:
```suggestion
preserving the high-level capabilities and ease-of-use of the |skycoord|
```
again, perhaps not the best if the goal is to get away from use like "easy", but I "efficiency" sometimes means "computational speed" in this context and here we really mean things like "easy to type", "you can use tab complete to find things", or "means you don't have to do a bunch of imports at the top".